### PR TITLE
Disable 'build:docs' for 'property-proxy' package

### PR DESCRIPTION
**Pull request by DLehenbauer**
Currently, 'build:docs' fails due to missing dependencies in package.json (fixed).

However, after adding the dependencies, running 'build:docs' produced an extraneous 'tsdoc-metadata.json' in the project root instead of the '/dist' directory.

As I started digging into why, I found that the other packages under '/experimental/' have 'build:docs' disabled.

### CODE REVIEW

File: experimental/PropertyDDS/packages/property-proxy/package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "build": "npm run build:genver && concurrently npm:build:compile npm:lint && npm run build:docs",
     "build:commonjs": "npm run tsc && npm run build:test",
     "build:compile": "concurrently npm:build:commonjs npm:build:esnext",
-    "build:docs": "api-extractor run --local && copyfiles -u 1 ./_api-extractor-temp/doc-models/* ../../../_api-extractor-temp/",
+    "build:docs": "",
```
Comment (curtisman) [inline experimental/PropertyDDS/packages/property-proxy/package.json:19]: can we just delete it for now?

Comment (curtisman) [reply experimental/PropertyDDS/packages/property-proxy/package.json:19]: or open an issue for it to fix and put in something similar to what is in `merge-tree`?

Comment (tylerbutler) [reply experimental/PropertyDDS/packages/property-proxy/package.json:19]: I would prefer to just remove this as well.

File: experimental/PropertyDDS/packages/property-proxy/package.json [hunk 2]
```diff
@@ -41,11 +41,13 @@
     "@babel/preset-env": "^7.2.0",
     "@fluidframework/build-common": "^0.22.0",
     "@fluidframework/eslint-config-fluid": "^0.23.0",
+    "@microsoft/api-extractor": "^7.16.1",
     "@types/jest": "22.2.3",
     "@typescript-eslint/eslint-plugin": "~4.14.0",
     "@typescript-eslint/parser": "~4.14.0",
     "babel-loader": "^8.0.5",
     "concurrently": "^5.2.0",
+    "copyfiles": "^2.1.0",
```
Comment (tylerbutler) [inline experimental/PropertyDDS/packages/property-proxy/package.json:50]: Might as well not include these deps since they're not needed.

**Review by tylerbutler — APPROVED** [review]

**Comment by DLehenbauer** [comment]
I removed build:doc (+ dependencies) and fixed the non-compliant build scripts.

Thanks @curtisman and @tylerbutler!

**Pull request merged**